xalia/main.gudl

```diff
@@ -604,7 +604,7 @@ if (role.(filler or internal_frame or panel or html_container or
               or info_bar or page_tab or page_tab_list or menu_bar or menu or
               combo_box or menu_item or pane or group or document or tree_table or
               toggle_button or list_box or list or table or title_bar or
-              tree)) {
+              tree or icon or small_icon or tile)) {
     recurse: true;

```

Comment:
These are not necessarily the roles of an element itself, but a particular type of a win32 listview element, correct? I could see this being an issue for other back ends, i.e at-spi2 has a `ATSPI_ROLE_ICON` which I would assume should have the role `icon`, but is probably not considered a container.

Reply:
Yeah, the role should probably be icon_view or something.

Reply:
Looking at CrossOver in Accerciser, it seems it uses layered_pane for the container and icon for the items.

Reply:
I'm not arguing, just looking for another container role I can use. I ended up changing the roles in the list view provider itself. How do you feel about it now.

Reply:
Looks good to me. :)